### PR TITLE
Week 1 2018

### PR DESCRIPTION
**Pull request by Dallas-D**
Updates training slides for week 1. Adds a git cheat sheet.

### CODE REVIEW

**Comment by ebretl** [comment]
don't mention in bash slide how everything is a file that is arranged in a tree structure. Just have them think in terms of folders

**Comment by ebretl** [comment]
Don't worry about chaining commands

**Comment by ebretl** [comment]
Add demo section -- opening terminal (nontrivial on Windows; install Git bash), how to enter these commands, general use case examples for commands that are important enough

**Comment by ebretl** [comment]
cheat sheet for terminal

**Comment by ebretl** [comment]
change git slide 4 to emphasize the usefulness of the search bar

**Comment by ebretl** [comment]
include way more diagrams for git concepts. Move "pull requests" diagram way earlier

**Comment by ebretl** [comment]
move installation instructions (e.g. git bash for windows) to beginning of presentation

**Comment by ebretl** [comment]
Add more emphasis on git status in slides

**Comment by ebretl** [comment]
show examples of adding specific files with add, like "git add folder/foo.cpp"

**Comment by ebretl** [comment]
Replace "remotes vs branches vs forks" slide with more diagrams in earlier slides

**Comment by ebretl** [comment]
Remove git as a graph slide

**Comment by ebretl** [comment]
slides from "creating branches" onward should have gitk (or similar visualization) screenshots of the effects of each step on a mostly emptry repo

File: slides/week1/week1.org [hunk 1]
```diff
@@ -22,9 +22,15 @@
...
-- (Almost) everything is a file
-- Every file is part of one big tree
-- Lots of tiny tools you can chain together to do useful things
+- (Almost) everything is a file or a folder
+- It lets you navigate your folders and manipulate files quickly
+- There are lots of tiny tools and commands you can use to do useful things
+
+* ATTENTION WINDOWS USERS
+- Terminal is not the same as Command Prompt
+- We do not recommend using CMD as it has different commands than Linux
```
Comment (JasonGibson274) [inline slides/week1/week1.org:31]: They *cannot* use cmd

File: slides/week1/week1.org [hunk 2]
```diff
@@ -1,130 +1,36 @@
...
+- Instead use Git bash
+  - *TODO:* Add install/use instructions for git bash
```
Comment (JasonGibson274) [inline slides/week1/week1.org:33]: Link to a website. https://www.atlassian.com/git/tutorials/install-git

Comment (JasonGibson274) [reply slides/week1/week1.org:33]: You do this later. Install with Git slides

**Review by JasonGibson274 — DISMISSED** [review]
Two things on general slides

File: slides/week1/week1.org (hunk 1 above)
Comment (Dallas-D) [reply slides/week1/week1.org:31]: I will change the phrasing of that line but I am interested in knowing why they can't. I would like to point out that I went through all of software training last year using command prompt

**Review by ebretl — APPROVED** [review]

**Pull request merged**